docs/ABOUT.md

```diff
@@ -1,4 +1,6 @@
-[OCaml](https://ocaml.org/) is an industrial strength programming language supporting functional, imperative and object-oriented styles. 
-It is used as a "teaching language" by a lot of institutions. 
-OCaml has a rich and powerful [type system](https://en.wikipedia.org/wiki/Type_system) that allows the compiler to catch some mistakes early on saving the developer a huge amount of frustration. 
-[Type inference](https://en.wikipedia.org/wiki/Type_inference) relieves the developer from the tedium of providing all the type signatures by themselves.
+[OCaml](https://ocaml.org/) is an industrial strength programming language supporting functional, imperative and object-oriented styles - but don't let this put you off, as it's used as a "teaching language" by a lot of institutions. 
+
+One of the best features of OCaml is the rich and powerful [type system](https://en.wikipedia.org/wiki/Type_system) - this is useful  to catch some mistakes early on saving developers a huge amount of frustration. 
+Also, [type inference](https://en.wikipedia.org/wiki/Type_inference) relieves developers from having to specify types in the code - the compiler will work these out for you!
+
+It's used by Facebook (for static code analysis with Infer) and Jane Street (for providing strong guarantees for their internal trading systems), and is the basis for the F# functional programming language.
```

Comment:
Also reasonml. Basis isn't quite the right word for f#, it's more like an influence.